RELEASE_GUIDE.md

```diff
@@ -42,50 +70,134 @@ considering what effects this may have:
 - JSON produced by `purs publish`
   - this might affect Pursuit
 
+Collaborate with other maintainers to identify what work needs to be done to resolve any issue.
+
 ## Making a release
 
-- Make a commit bumping versions. The following should be updated:
+To make a release, a maintainer should create a PR updating versions in the repo, and once that is merged, they can create a GitHub release, which triggers a CI build that should upload prebuilt binaries into that GitHub release page. (If it doesn't, you'll upload those manually).
```

Comment:
You've mentioned this both at the start and the end of this section. I don't think it's necessary to mention it twice; can we remove one?